deploy/examples/operator.yaml

```
@@ -149,22 +149,34 @@ data:
   #   - key: node.rook.io/rbd
   #     operator: Exists
 
-  # (Optional) CephCSI CephFS provisioner NodeAffinity(if specified, overrides CSI_PROVISIONER_NODE_AFFINITY).
+  # (Optional) CephCSI CephFS provisioner NodeAffinity (if specified, overrides CSI_PROVISIONER_NODE_AFFINITY).
   # CSI_CEPHFS_PROVISIONER_NODE_AFFINITY: "role=cephfs-node"
   # (Optional) CephCSI CephFS provisioner tolerations list(if specified, overrides CSI_PROVISIONER_TOLERATIONS).
   # Put here list of taints you want to tolerate in YAML format.
   # CSI provisioner would be best to start on the same nodes as other ceph daemons.
   # CSI_CEPHFS_PROVISIONER_TOLERATIONS: |
   #   - key: node.rook.io/cephfs
   #     operator: Exists
-  # (Optional) CephCSI CephFS plugin NodeAffinity(if specified, overrides CSI_PLUGIN_NODE_AFFINITY).
+  # (Optional) CephCSI CephFS plugin NodeAffinity (if specified, overrides CSI_PLUGIN_NODE_AFFINITY).
   # CSI_CEPHFS_PLUGIN_NODE_AFFINITY: "role=cephfs-node"
   # (Optional) CephCSI CephFS plugin tolerations list(if specified, overrides CSI_PLUGIN_TOLERATIONS).
   # Put here list of taints you want to tolerate in YAML format.
   # CSI plugins need to be started on all the nodes where the clients need to mount the storage.
   # CSI_CEPHFS_PLUGIN_TOLERATIONS: |
   #   - key: node.rook.io/cephfs
   #     operator: Exists
+  # NOTE: Support for defining NodeAffinity for operators other than "In" and "Exists" requires the user to input a
+  # valid v1.NodeAffinity JSON or YAML string. For example, the following is a valid YAML v1.NodeAffinity string:
+  # `
```

In my previous comment about formatting I was suggesting:
- If they want to use the example, all they should need to do is remove the # marks. All other formatting should be valid
- Add an example env var where this can be used, such as `CSI_CEPHFS_PLUGIN_NODE_AFFINITY`. Currently these comments are included after the tolerations
- Use the `|` for a multi-line string instead of tick marks. This is just yaml, not markup. Tick marks won't work in the yaml, right? 
- No need for `---` since it's just a single yaml snippet, right?
- For compactness, the `-` doesn't need its own line. For example: `- matchExpressions` can be a single line

```suggestion
  # CSI_CEPHFS_PLUGIN_NODE_AFFINITY: |
```

---

> Add an example env var where this can be used, such as CSI_CEPHFS_PLUGIN_NODE_AFFINITY. Currently these comments are included after the tolerations

Moved it below `CSI_CEPHFS_PLUGIN_NODE_AFFINITY` for more relevance.

> Use the | for a multi-line string instead of tick marks. This is just yaml, not markup. Tick marks won't work in the yaml, right?
No need for --- since it's just a single yaml snippet, right?
For compactness, the - doesn't need its own line. For example: - matchExpressions can be a single line

Modified.